synapse/lib/stormhttp.py

```diff
@@ -116,18 +129,29 @@ async def _httpRequest(self, meth, url, headers=None, json=None, body=None, ssl_
             except asyncio.CancelledError:  # pragma: no cover
                 raise
             except Exception as e:
-                mesg = f'Error during http {meth} - {str(e)}'
-                raise s_exc.StormRuntimeError(mesg=mesg, headers=headers, json=json, body=body, params=params) from None
+                logger.exception(f'Error during http {meth} @ {url}')
+                errname, erfo = s_common.err(e)
+                erfo['errname'] = errname
+                info = {
+                    'code': -1,
+                    'headers': dict(),
+                    'url': url,
+                    'body': b'',
+                    'erfo': erfo,
```

Comment:
lets break these out as `errname` and `errinfo` so they're a bit more consistent with some of our other exception serialization

Reply:
willing to `erfo` -> `errinfo` too? `erfo` sounds a bit too much like a `tufo` construct and we've got `errinfo` in `s_common.excinfo()` :D